### PR TITLE
[MM-37639]: adding test case MM-T4446

### PR DESCRIPTION
**Pull request by michelengelen**
#### Summary
adding in test case MM-T4446 for deleting a reply in a thread and assert visibility in global threads view

I have set e2e/MM-T4445 as base branch (#10050) since it relies on some stuff in there. 
This will probably the case for MM-T4447, MM-T4448 and MM-T4379 as well.

#### Ticket Link
[MM-37693](https://mattermost.atlassian.net/browse/MM-37693)

#### Related Pull Requests
n/a

#### Screenshots
n/a

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/tests/integration/collapsed_reply_threads/root_post_spec.js [hunk 1]
```diff
@@ -84,4 +99,49 @@ describe('Collapsed Reply Threads', () => {
         // * There should be no thread item anymore
         cy.get('article.ThreadItem').should('have.lengthOf', 0);
     });
+
+    it('MM-T4446 delete single reply on thread (current behavior is incorrect, see comments)', () => {
```
Comment (saturninoabril) [inline e2e/cypress/tests/integration/collapsed_reply_threads/root_post_spec.js:103]: I suggest to match the title with Zephyr. So either update here or in Zephyr.
```suggestion
    it('MM-T4446 CRT - Delete single reply post on a thread', () => {
```

File: e2e/cypress/tests/integration/collapsed_reply_threads/root_post_spec.js [hunk 2]
```diff
@@ -84,4 +99,49 @@ describe('Collapsed Reply Threads', () => {
...
+        /**
+         * When you delete a reply you are still following the thread, so the thread is not being removed
+         * from the global threads view. The test description was written differently asit was assuming the
```
Comment (saturninoabril) [inline e2e/cypress/tests/integration/collapsed_reply_threads/root_post_spec.js:106]:
```suggestion
         * from the global threads view. The test description was written differently as it was assuming the
```

**Review by saturninoabril — APPROVED** [review]
Thanks @michelengelen! LGTM except for couple requests.

E2E at https://automation-dashboard.vercel.app/cycles/fadd82c3-5267-482c-b613-10cee505c417

**Comment by michelengelen** [comment]
FYI @saturninoabril @jgilliam17 I will close this one and continue working on the feature branch ... the following test cases all build up on this one and handling them in a slightly bigger PR is probably easier since it will not introduce merge conflicts later on.

Also, as @metanerd pointed out, testing e2e with the slash command is not supported for PRs comparing against feature branches.

**Pull request closed without merging**